src/device_trezor/trezor/messages/messages-common.pb.cc

```diff
@@ -0,0 +1,3357 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
+// source: messages-common.proto
```

Comment:
It depends. Without the generated messages it won't compile, the generation is performed by cmake at the moment and you need to have protobuf compiler installed (and the submodule trezor-common).

Do you prefer to remove it from the versioning and leave it up to the cmake to generate it just before the build? Is then this directory OK for the generated files?

Reply:
My preference is for generated files to not be in the repo unless they're hard/impractical to generate (like the checkpoints.dat file, which requires an existing blockchain, and is manual since it'd change all the time).

Is "protobuf compiler" hard to install/run ?

Reply:
> The commit message seems to have some long history of all wip commits, is that intended ?

I keep it here to keep track of squashed commits. A can remove it after we are done with the review if that is OK for you.